query-engine/connectors/sql-query-connector/src/sql_info.rs

```diff
@@ -77,7 +77,7 @@ impl SqlInfo {
         Self {
             family: SqlFamily::Postgres,
             max_rows: None,
-            max_bind_values: get_batch_size(32767),
+            max_bind_values: get_batch_size(32766),
```

Comment:
I'd be safe and make it much smaller. 25000 should be enough for everybody.

Reply:
Ok, fair enough

Reply:
I advocated for just fixing the currently most popuplar case here, and then use future reports to understand where else our chunking logic fails and leads to the error being thrown. If we just go to a lower number, we will never be able to find these and forever just create more queries than stricly necessary (at the cost of performance).